features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -1388,6 +1386,16 @@ internal open class RumViewScope(
             interactionToNextViewMetricResolver: InteractionToNextViewMetricResolver,
             networkSettledResourceIdentifier: InitialResourceIdentifier
         ): RumViewScope {
+            val networkSettledMetricResolver = NetworkSettledMetricResolver(
+                networkSettledResourceIdentifier,
+                sdkCore.internalLogger
+            )
+
+            val viewEndedMetricDispatcher = ViewEndedMetricDispatcher(
+                viewType = RumViewType.FOREGROUND,
+                internalLogger = sdkCore.internalLogger
+            )
```

Comment:
Is it a bold assumption that `RumViewScope#fromEvent` will always be called for foreground flow?

Reply:
Yes, basically it's default value for the RumViewScope. I will pass it explicitly